metatron-app/metatron-discovery
# F#1644 uri type dataset support

### Description

Now we support file datasets of URI type.
The URI type will work much like the upload type.

**Related Issue** : 

[#1644](https://github.com/metatron-app/metatron-discovery/issues/1644)

### How Has This Been Tested?

1. create a dataset of URI type and see the details
2. create a wrangled dataset and see the details
3. create a snapshot and see the details
It should work well in the whole process.

### Types of changes



### Checklist:



### Additional Context
